lua/ulx/modules/sh/cfc_decals.lua

```diff
@@ -0,0 +1,33 @@
+CFCUlxCommands.decals = CFCUlxCommands.decals or {}
+local cmd = CFCUlxCommands.decals
+
+local entsToClear = {
+    ["class C_ClientRagdoll"] = true,
+}
+
+if CLIENT then
+    net.Receive( "ulx_clean_decals", function()
+        for _, ent in pairs( ents.GetAll() ) do
+            if entsToClear[ent:GetClass()] then
+                ent:Remove()
+            end
+        end
+    end )
+else
+    util.AddNetworkString( "ulx_clean_decals" )
+end
+
+function cmd.decals( callingPlayer )
+    for _, ply in ipairs( player.GetHumans() ) do
+        ply:ConCommand( "r_cleardecals" )
+
+        net.Start( "ulx_clean_decals" )
+        net.Broadcast()
```

Comment:
`net.Broadcast` already runs on all players, you'll need to take it out of that loop 😬